### DISCUSSION TITLE
Possible to show what languages you have completed a kata in?

### DISCUSSION THREAD

**Discussion by stuartstein777** [codewars/codewars.com#2365] [Q&A] [answered]
Would it be possible to highlight on the kata selection screen what languages you have completed a kata in?

The reason i ask is I solve in lots of languages, but mostly Clojure. It would be nice to be able to see Kata that I have solved in other languages but not solved in Clojure.

e.g, something like this:
![codwars](https://user-images.githubusercontent.com/22107263/116785655-a9769480-aa92-11eb-9a12-e0cc82e6de77.png)

Alternatively maybe add filter to filter out kata I have completed but not in a certain language?

**Comment by kazk** [comment, marked as answer]
No, it's not possible at the moment. See https://github.com/codewars/codewars.com/issues/1162

I want this too. But this requires a fundamental change to how the search page is rendered/cached on the server to be able to filter by language and display each status, so I haven't had enough time to make the changes.

Another approach is to persist the records of your completions on the client side and apply styles to the search results based on it. This won't allow you to filter the search, but it's better than what we have. If you don't mind using an unofficial userscript, see https://github.com/codewars/codewars.com/issues/1758. The author is one of our moderators now, so I trust him, but I don't use it, and haven't reviewed the code.
